src/shell/Environment.ts

```diff
@@ -34,7 +34,7 @@ export const preprocessEnv = (lines: string[]) => {
 
 export const processEnvironment: Dictionary<string> = {};
 export async function loadEnvironment(): Promise<void> {
-    const lines = preprocessEnv(await executeCommandWithShellConfig("env"));
+    const lines = preprocessEnv(await executeCommandWithShellConfig("set"));
```

Comment:
⚠ Warning: Unix-specific command was replaced by a Windows-specific; need to find a portable way to express this command.